docs/generate_emmylua.py

```diff
@@ -302,6 +302,7 @@ def main():
                 if "comment" in var and var["comment"]
                 else ""
             )
+            comment_str = comment_str.replace("`", "");
```

Comment:
I'm pretty sure code blocks work well in LSPs hints and stuff

Reply:
in that case, we should probably not remove them in other places

Reply:
Reverted this change, don't really want to mess with the other code that removed the code blocks in other comments